### PR TITLE
quickstart image fix

### PR DESCRIPTION
**Pull request by ddematheu2**
# What
fixed images on quickstarts

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

**Review by alcail — APPROVED** [review]

File: packages/storybook/stories/QuickStarts/QuickstartComposite.stories.mdx [hunk 1]
```diff
@@ -25,6 +25,18 @@ You can find the completed code for this quickstart here: [Get started with Comp
...
 - An identity with both VoIP and Chat scopes. Generate an identity using the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/identity/quick-create-identity).
 
+## Generate Access Token
+
+For setting up adapters that power the composites, we need to generate an access token to allow the adapters to interact with the Azure Communication Services back-end.
+
+Access Tokens can be generated using this [Azure Portal quickstart](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/identity/quick-create-identity) inside your Azure Communication Services dashboard as shown below.
+
+<Stack style={{ width: '100%' }}>
+  <video style={{ width: '100%' }} src="videos/generate-access-token.mp4" autoPlay="true" loop="true"></video>
```
Comment (prprabhu-ms) [inline packages/storybook/stories/QuickStarts/QuickstartComposite.stories.mdx:35]:
```suggestion
  <video style={{ width: '100%' }} src="videos/generate-access-token.mp4" autoPlay loop muted></video>
```

File: packages/storybook/stories/QuickStarts/QuickstartStatefulCall.stories.mdx [hunk 2]
```diff
@@ -30,6 +31,18 @@ You can find the completed code for this quickstart here: [Get started with Call
...
 - Knowledge of Azure Communication Services [Call Concepts](https://docs.microsoft.com/azure/communication-services/concepts/voice-video-calling/about-call-types).
 
+## Generate Access Token
+
+For setting up adapters that power the composites, we need to generate an access token to allow the adapters to interact with the Azure Communication Services back-end.
+
+Access Tokens can be generated using this [Azure Portal quickstart](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/identity/quick-create-identity) inside your Azure Communication Services dashboard as shown below.
+
+<Stack style={{ width: '100%' }}>
+  <video style={{ width: '100%' }} src="videos/generate-access-token.mp4" autoPlay="true" loop="true"></video>
```
Comment (prprabhu-ms) [inline packages/storybook/stories/QuickStarts/QuickstartStatefulCall.stories.mdx:41]:
```suggestion
  <video style={{ width: '100%' }} src="videos/generate-access-token.mp4" autoPlay loop muted></video>
```

File: packages/storybook/stories/QuickStarts/QuickstartStatefulChat.stories.mdx [hunk 3]
```diff
@@ -31,6 +32,18 @@ You can find the completed code for this quickstart here: [Get started with Chat
...
   You will need to create a thread and a user to that thread to use the UI Library. Look at the [recommended architecture for the UI Library](./?path=/docs/usecases--page#recommended-architecture)
 
+## Generate Access Token
+
+For setting up adapters that power the composites, we need to generate an access token to allow the adapters to interact with the Azure Communication Services back-end.
+
+Access Tokens can be generated using this [Azure Portal quickstart](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/identity/quick-create-identity) inside your Azure Communication Services dashboard as shown below.
+
+<Stack style={{ width: '100%' }}>
+  <video style={{ width: '100%' }} src="videos/generate-access-token.mp4" autoPlay="true" loop="true"></video>
```
Comment (prprabhu-ms) [inline packages/storybook/stories/QuickStarts/QuickstartStatefulChat.stories.mdx:42]:
```suggestion
  <video style={{ width: '100%' }} src="videos/generate-access-token.mp4" autoPlay loop muted></video>
```

**Review by prprabhu-ms — APPROVED** [review]

**Comment by prprabhu-ms** [comment]
Let's land this, to close https://skype.visualstudio.com/SPOOL/_queries/edit/2657899/?

**Pull request merged**